<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmColorButton</class>
 <widget class="QWidget" name="frmColorButton">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QWidget" name="widgetBtn" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="ColorButton" name="colorButton1" native="true"/>
      </item>
      <item row="0" column="1">
       <widget class="ColorButton" name="colorButton2" native="true"/>
      </item>
      <item row="0" column="2">
       <widget class="ColorButton" name="colorButton3" native="true"/>
      </item>
      <item row="0" column="3">
       <widget class="ColorButton" name="colorButton4" native="true"/>
      </item>
      <item row="0" column="4">
       <widget class="ColorButton" name="colorButton5" native="true"/>
      </item>
      <item row="1" column="0">
       <widget class="ColorButton" name="colorButton6" native="true"/>
      </item>
      <item row="1" column="1">
       <widget class="ColorButton" name="colorButton7" native="true"/>
      </item>
      <item row="1" column="2">
       <widget class="ColorButton" name="colorButton8" native="true"/>
      </item>
      <item row="1" column="3">
       <widget class="ColorButton" name="colorButton9" native="true"/>
      </item>
      <item row="1" column="4">
       <widget class="ColorButton" name="colorButton10" native="true"/>
      </item>
      <item row="2" column="0">
       <widget class="ColorButton" name="colorButton11" native="true"/>
      </item>
      <item row="2" column="1">
       <widget class="ColorButton" name="colorButton12" native="true"/>
      </item>
      <item row="2" column="2">
       <widget class="ColorButton" name="colorButton13" native="true"/>
      </item>
      <item row="2" column="3">
       <widget class="ColorButton" name="colorButton14" native="true"/>
      </item>
      <item row="2" column="4">
       <widget class="ColorButton" name="colorButton15" native="true"/>
      </item>
      <item row="3" column="0">
       <widget class="ColorButton" name="colorButton16" native="true"/>
      </item>
      <item row="3" column="1">
       <widget class="ColorButton" name="colorButton17" native="true"/>
      </item>
      <item row="3" column="4">
       <widget class="ColorButton" name="colorButton20" native="true"/>
      </item>
      <item row="3" column="2">
       <widget class="ColorButton" name="colorButton18" native="true"/>
      </item>
      <item row="3" column="3">
       <widget class="ColorButton" name="colorButton19" native="true"/>
      </item>
      <item row="4" column="0">
       <widget class="ColorButton" name="colorButton21" native="true"/>
      </item>
      <item row="4" column="1">
       <widget class="ColorButton" name="colorButton22" native="true"/>
      </item>
      <item row="4" column="2">
       <widget class="ColorButton" name="colorButton23" native="true"/>
      </item>
      <item row="4" column="3">
       <widget class="ColorButton" name="colorButton24" native="true"/>
      </item>
      <item row="4" column="4">
       <widget class="ColorButton" name="colorButton25" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ColorButton</class>
   <extends>QWidget</extends>
   <header>colorbutton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
